dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/AbusiveUserMapper.java

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.dynamometer.workloadgenerator;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hdfs.DistributedFileSystem;
+import org.apache.hadoop.io.LongWritable;
+import org.apache.hadoop.io.NullWritable;
+import org.apache.hadoop.security.UserGroupInformation;
+
+import java.io.IOException;
+import java.net.URI;
+import java.security.PrivilegedAction;
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+
+/**
+ * AbusiveUserMapper consists of two types of users: one type that performs a large listing operation on a directory
+ * with many files, and another type that performs a small operation (either a single file listing or a single directory
+ * creation). This uses {@link TimedInputFormat}; see its Javadoc for configuration information. Only requires the
+ * NameNode, does not make file data changes.
+ *
+ * <p>Configuration options available:
```

Comment:
I feel like the wording is a bit weird on some of these descriptions, suggestions welcome!